### PR TITLE
Add test edge cases for String.times(Int)

### PR DESCRIPTION
**Pull request by Goooler**
Amend for #54.

### CODE REVIEW

File: base/src/test/kotlin/io/goooler/demoapp/base/BaseExtensionsTest.kt [hunk 1]
```diff
@@ -3,11 +3,18 @@ package io.goooler.demoapp.base
...
 
   @Test
   fun `String times(Int)`() {
     assertEquals("1" * 3, "111")
+    assertEquals("1" * 0, "")
+    assertThrows<IllegalArgumentException> {
+      "1" * -1
+    }.message.let {
+      assertEquals(it, "Param num should >= 0")
+    }
```
Comment (Goooler) [inline base/src/test/kotlin/io/goooler/demoapp/base/BaseExtensionsTest.kt]:
```kotlin
val exception = assertThrows<IllegalArgumentException> {
    throw IllegalArgumentException("Talk to a duck")
}
assertEquals("Talk to a duck", exception.message)

```

File: base/src/test/kotlin/io/goooler/demoapp/base/BaseExtensionsTest.kt [hunk 2]
```diff
@@ -2,12 +2,19 @@ package io.goooler.demoapp.base
...
 
 class BaseExtensionsTest {
 
   @Test
   fun `String times(Int)`() {
     assertEquals("1" * 3, "111")
+    assertEquals("1" * 0, "")
+    assertFailsWith<IllegalArgumentException> {
+      "1" * -1
+    }.message.let {
```
Comment (Goooler) [inline base/src/test/kotlin/io/goooler/demoapp/base/BaseExtensionsTest.kt:16]: https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-fails-with.html
